internal/generator/fluentd/conf_test.go

```diff
@@ -410,7 +410,7 @@ var _ = Describe("Testing Complete Config Generation", func() {
     kubernetes_url 'https://kubernetes.default.svc'
     annotation_match ["^containerType\.logging\.openshift\.io\/.*$"]
     cache_size '1000'
-    watch 'false'
+    watch true
```

Comment:
default value of watch is `true`
https://github.com/ViaQ/logging-fluentd/blob/main/fluentd/vendored_gem_src/fluent-plugin-kubernetes_metadata_filter/lib/fluent/plugin/filter_kubernetes_metadata.rb#L48